modules/031-local-path-provisioner/docs/USAGE_RU.md

```diff
@@ -14,3 +14,10 @@ spec:
   - system
   path: "/opt/local-path-provisioner"
 ```
+
+Примечания:
+
+- Этот пример создаст `localpath-system` класс (`storage class`) который **должен** быть использован в pod'ах что бы все заработало
+- Все создаваемые хранилища будут иметь политику очистки `Delete` ([issue](https://github.com/deckhouse/deckhouse/issues/360))
+- Если этот объект будет удален раньше чем объекты его использующие, папки с сервера удалены не будут
+- Обратите внимание - в примере предпологается создание папок на системных нодах, которые скорее всего имеют ряд ограничителей (taints), а как следствие pod'ы **должны** иметь соотв. tolerations
```

Comment:
this isn't needed due to prometheus has tolerations for system nodes by default.